requirements-aws.txt

```diff
@@ -0,0 +1,2 @@
+amazon-braket-sdk
+qiskit-braket-provider
```

Comment:
is this file needed at all?

Reply:
`amazon-braket-sdk` is needed but `qiskit-braket-provider` is not. I will remove the qiskit dependencies.

Reply:
This file is not mentioned in any other file, and the dependencies have been added to `pyproject.toml`.

What would you need this file for?

Reply:
Hi @alecandido , do you mean the `amazon-braket-sdk`? 

If I'm not wrong, any `braket` command such as `from braket.devices import LocalSimulator` and `from braket.circuits import Circuit as BraketCircuit` requires `amazon-braket-sdk`. It is listed as a prerequesite here: https://github.com/amazon-braket/amazon-braket-sdk-python?tab=readme-ov-file#prerequisites.

I've also mentioned in the documentation under "Registering for an AWS Account" in the doc/source/tutorials.rst file.

Reply:
No, what we mean is that the whole file `requirements-aws.txt` is probably not needed, as the requirements (dependencies) are specified in the `pyproject.toml` already.

Reply:
I see, sorry for misunderstanding.

Pinging @TL231! Tan Le, can I trouble you to check if we can remove the `requirements-aws.txt` file altogether? I think we wrote this at the very start to ensure that the user had the right packages installed. Thanks a lot!